src/platforms/python/guides/pymongo/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: PyMongo
+description: "Learn about using Sentry with PyMongo."
+---
+
+The PyMongo integration adds support for [PyMongo](https://www.mongodb.com/docs/drivers/pymongo/), the official MongoDB
+driver. It adds breadcrumbs and performace traces for all queries.
+
+## Install
+
+Install `sentry-sdk` from PyPI with the `pymongo` extra:
+
+```bash
+pip install --upgrade 'sentry-sdk[pymongo]'
+```
+
+## Configure
+
+To configure the SDK, initialize it before creating any of PyMongo's MongoClient instances:
+
+```python
+import sentry_sdk
+from sentry_sdk.integrations.pymongo import PyMongoIntegration
+
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    integrations=[
+        PyMongoIntegration(),
+    ],
+
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    # We recommend adjusting this value in production,
+    traces_sample_rate=1.0,
+)
+```
+
+<Note>
+
+In the future, we will make this auto-enabling, so you won't need to add the `PyMongoIntegration` to your
+`sentry_sdk.init()` call. Instead, the Sentry SDK will detect the presence of PyMongo and will set up the integration
+automatically.
+
+</Note>
+
+## Other MongoDB libraries
+
+PyMongo is an official synchronous driver for MongoDB. It means that many other Python libraries interacting with
+MongoDB use it under the hood, like, for example `mongoengine` (always) or `umongo` (if selected from multiple available
+drivers). Queries generated by those libraries will also be monitored.
+
+If you use a different driver (for example, `motor`, `TxMongo`), this integration won't work.
```

Comment:
Hi,
we use `motor` in our project, and it also [uses](https://motor.readthedocs.io/en/stable/requirements.html) `pymongo` under the hood, so this integration will work (actually, I checked it by monkey patching the same integration some time ago, so I am quite sure in this)).

Reply:
Oh. Interesting. Didn't know it's built on top of pymongo. Expected it to be written from scratch. But indeed it depends on pymongo. Will update this fragment.